pyvista/plotting/plotting.py

```diff
@@ -849,9 +849,28 @@ def add_legend(self, *args, **kwargs):
 
     @wraps(Renderer.remove_legend)
     def remove_legend(self, *args, **kwargs):
-        """Wrap ``Renderer.remove_legend``."""
+        """Wrap ``Renderer.remove_legend``.
         return self.renderer.remove_legend(*args, **kwargs)
+        
+        Examples
+        --------
+        Add a legend and then remove it.
+        
+        >>> import pyvista
+        >>> from pyvista import examples
+        >>> sphere = pyvista.Sphere(center=(0, 0, 1))
+        >>> cube = pyvista.Cube()
+        >>> pl = pyvista.Plotter()
+        >>> _ = pl.add_mesh(sphere, 'grey', smooth_shading=True, label='Sphere')
+        >>> _ = pl.add_mesh(cube, 'r', label='Cube')
+        >>> _ = pl.add_legend(bcolor='w', face=None)
+        >>> pl.show()
 
+        >>> pl.remove_legend()
```

Comment:
It's not possible to close and then reopen a plotter. You will probably just want to:

```suggestion
        >>> pl.remove_legend()
```

Reply:
@akaszynski Thanks. @ENIAC-6  Could you click `Add suggestion to batch` and `commit changes` ?